packages/driver/src/cypress/state.ts

```diff
@@ -36,7 +36,6 @@ export interface StateFunc {
   (k: 'canceld', v?: boolean): boolean
   (k: 'error', v?: Error): Error
   (k: 'assertUsed', v?: boolean): boolean
-  (k: 'currentAssertionUserInvocationStack', v?: string): string
```

Comment:
Should this have been removed? We still use it

Reply:
Ah, whoops. At one point in the PR it was unused; it got added back to the code but missed from the types here. Re-added.